### ISSUE TITLE
Problem med jq i Docker

### ISSUE THREAD

**Issue by LemurP** [kantega/Kantega-Code-Analyzer#1]
Lager og kjører Dockerimage fra rot:
`docker build -t kca . && docker run -it kca`
Kjører deretter `kca_init` uten problemer, men støter på problem med `kca_hotspots`

> Steg 2 Genererer kompleksitet fil
/app/scripts/../bin/nix/jq: error while loading shared libraries: libonig.so.5: cannot open shared object file: No such file or directory
thread 'main' panicked at library/std/src/io/stdio.rs:1118:9:
failed printing to stdout: Broken pipe (os error 32)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

Ser ut som at jq ikke klarer å finne noe den trenger, men jeg vet ikke hva det er.

@KantegaAndreas ser du hva dette skyldes?

**Comment by Northcode** [comment]
ja, antar du må installere https://packages.ubuntu.com/focal/amd64/libonig5/ i docker imaget.
`RUN apt install libonig5`
